packages/ember-glimmer/lib/template.js

```diff
@@ -1,17 +1,12 @@
 import { Template } from 'glimmer-runtime';
 
-let templateId = 0;
-
 class Wrapper {
-  static create(options) {
```

Comment:
Isn't this needed for container-based instantiation? (However, I do see that the tests are passing, and I'm not sure why)